<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RentAdd</class>
 <widget class="QWidget" name="RentAdd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>624</width>
    <height>451</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>624</width>
    <height>451</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>624</width>
    <height>451</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Наименование</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_6">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>С</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="startDate">
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::UpDownArrows</enum>
       </property>
       <property name="displayFormat">
        <string>dd/MM/yyyy</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>на</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="numberOfDays">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>365</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>сут.</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Время выдачи:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTimeEdit" name="startTime"/>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="0" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Состояние:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="status">
       <property name="minimumSize">
        <size>
         <width>196</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>196</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Пометка:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="note">
       <property name="maxLength">
        <number>200</number>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <layout class="QGridLayout" name="gridLayout_5">
     <item row="0" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Дата приемки:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QDateEdit" name="endDate">
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::UpDownArrows</enum>
       </property>
       <property name="displayFormat">
        <string>dd/MM/yyyy</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Время приемки:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QTimeEdit" name="endTime"/>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_9">
       <property name="minimumSize">
        <size>
         <width>105</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>105</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>Ф.И.О:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="fio">
       <property name="minimumSize">
        <size>
         <width>441</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>441</width>
         <height>20</height>
        </size>
       </property>
       <property name="maxLength">
        <number>250</number>
       </property>
       <property name="placeholderText">
        <string>Введите ФИО...</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_10">
       <property name="minimumSize">
        <size>
         <width>105</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>105</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>Паспорт:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="passport">
       <property name="minimumSize">
        <size>
         <width>441</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>441</width>
         <height>20</height>
        </size>
       </property>
       <property name="maxLength">
        <number>250</number>
       </property>
       <property name="placeholderText">
        <string>Введите серию и номер паспорта...</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_11">
       <property name="minimumSize">
        <size>
         <width>105</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>105</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>Вод.удостоверение:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="drivingLicense">
       <property name="minimumSize">
        <size>
         <width>441</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>441</width>
         <height>20</height>
        </size>
       </property>
       <property name="maxLength">
        <number>250</number>
       </property>
       <property name="placeholderText">
        <string>Введите номер водительского удостоверения...</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_12">
       <property name="minimumSize">
        <size>
         <width>105</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>105</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>Адрес:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="address">
       <property name="minimumSize">
        <size>
         <width>441</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>441</width>
         <height>20</height>
        </size>
       </property>
       <property name="maxLength">
        <number>250</number>
       </property>
       <property name="placeholderText">
        <string>Введите адрес...</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_13">
       <property name="minimumSize">
        <size>
         <width>105</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>105</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>Телефон:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="telephoneNumber">
       <property name="minimumSize">
        <size>
         <width>441</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>441</width>
         <height>20</height>
        </size>
       </property>
       <property name="maxLength">
        <number>250</number>
       </property>
       <property name="placeholderText">
        <string>Введите телефонный номер...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0" colspan="3">
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="3">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="label_14">
       <property name="minimumSize">
        <size>
         <width>105</width>
         <height>16</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>105</width>
         <height>16</height>
        </size>
       </property>
       <property name="text">
        <string>Залог:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QDoubleSpinBox" name="deposit">
       <property name="minimumSize">
        <size>
         <width>83</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>83</width>
         <height>20</height>
        </size>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="decimals">
        <number>2</number>
       </property>
       <property name="maximum">
        <double>1000000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_16">
       <property name="minimumSize">
        <size>
         <width>426</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>426</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>рублей</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_15">
       <property name="minimumSize">
        <size>
         <width>105</width>
         <height>16</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>105</width>
         <height>16</height>
        </size>
       </property>
       <property name="text">
        <string>Доп.инфо:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="addInfo">
       <property name="minimumSize">
        <size>
         <width>441</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>441</width>
         <height>20</height>
        </size>
       </property>
       <property name="maxLength">
        <number>250</number>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="6" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_17">
       <property name="minimumSize">
        <size>
         <width>105</width>
         <height>13</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>105</width>
         <height>13</height>
        </size>
       </property>
       <property name="text">
        <string>За сутки:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="dayPrice">
       <property name="minimumSize">
        <size>
         <width>131</width>
         <height>22</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="6" column="1">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="2">
      <widget class="QLabel" name="label_20">
       <property name="minimumSize">
        <size>
         <width>78</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>78</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>рублей</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_18">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>13</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>13</height>
        </size>
       </property>
       <property name="text">
        <string>Сумма:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_19">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>13</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>13</height>
        </size>
       </property>
       <property name="text">
        <string>Оплачено:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_21">
       <property name="minimumSize">
        <size>
         <width>78</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>78</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>рублей</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="alreadyPaid">
       <property name="minimumSize">
        <size>
         <width>83</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>83</width>
         <height>20</height>
        </size>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="maximum">
        <double>1000000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QDoubleSpinBox" name="totalSum">
       <property name="minimumSize">
        <size>
         <width>83</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>83</width>
         <height>20</height>
        </size>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="maximum">
        <double>1000000.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="6" column="2">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="rentAddOkB">
       <property name="text">
        <string>ОК</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="rentAddCancelB">
       <property name="text">
        <string>Отмена</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>startDate</tabstop>
  <tabstop>numberOfDays</tabstop>
  <tabstop>startTime</tabstop>
  <tabstop>status</tabstop>
  <tabstop>endDate</tabstop>
  <tabstop>note</tabstop>
  <tabstop>endTime</tabstop>
  <tabstop>fio</tabstop>
  <tabstop>passport</tabstop>
  <tabstop>drivingLicense</tabstop>
  <tabstop>address</tabstop>
  <tabstop>telephoneNumber</tabstop>
  <tabstop>deposit</tabstop>
  <tabstop>addInfo</tabstop>
  <tabstop>dayPrice</tabstop>
  <tabstop>totalSum</tabstop>
  <tabstop>alreadyPaid</tabstop>
  <tabstop>rentAddOkB</tabstop>
  <tabstop>rentAddCancelB</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
